dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/context/ConfigManager.java

```diff
@@ -310,7 +310,7 @@ public void refreshAll() {
     }
 
     private void checkDuplicate(AbstractConfig oldOne, AbstractConfig newOne) {
-        if (oldOne != null && !oldOne.equals(newOne)) {
+        if (oldOne != null && oldOne.equals(newOne)) {
```

Comment:
I notice the UT is missing here, could you add a unit test for this change?

Reply:
ok

Reply:
please check `AbstractInterfaceConfigTest#checkDuplicateApplication`